src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -0,0 +1,401 @@
+﻿namespace Equinox.MessageDb
+
+open System
```

Comment:
for better or worse I do alphabetic for ease of merge conflits rather than topological grouping.